block-node/app/docker/Dockerfile

```
@@ -71,7 +71,7 @@ RUN groupadd --gid 2000 hedera && \
 ARG VERSION
 
 # Set the working directory inside the container
-WORKDIR /app
+WORKDIR /opt/hiero/block-node
```

Consider an ARG
```suggestion
ARG BN_WORKDIR="/opt/hiero/block-node"
WORKDIR ${BN_WORKDIR}
```

Then use ${BN_WORKDIR} in place of string

---

Thanks for the suggestion, I refactored both block-node/app/docker/Dockerfile and simulator/docker/Dockerfile. In the first dockerfile I promoted ARG to ENV in the "Build Final Image" section in order to access the value in the ENTRYPOINT command properly. The path is not a sensitive information so it should not cause any issues.